pycoal/tests/test.py

```diff
@@ -26,7 +26,7 @@ def _remove_files(listOfFileNames):
             pass
 
 # file names of USGS Digital Spectral Library 06 in ENVI format
-libraryFilenames = ["s06av95a_envi.hdr", "s06av95a_envi.sli"]
+libraryFilenames = ["s07_AV95_envi.hdr", "s07_AV95_envi.sli"]
```

Comment:
... and again. You need to add new entries not take away from what we already have.